Solutions/CiscoSEG/Data Connectors/Connector_Cisco_SEG_CEF.json

```diff
@@ -0,0 +1,127 @@
+{
+    "id": "CiscoSEG",
+    "title": "Cisco Secure Email Gateway",
+    "publisher": "Cisco",
+    "descriptionMarkdown": "The [Cisco Secure Email Gateway (SEG)](https://www.cisco.com/c/en/us/products/security/email-security/index.html) data connector provides the capability to ingest [Cisco SEG Consolidated Event Logs](https://www.cisco.com/c/en/us/td/docs/security/esa/esa14-0/user_guide/b_ESA_Admin_Guide_14-0/b_ESA_Admin_Guide_12_1_chapter_0100111.html#con_1061902) into Azure Sentinel.",
+    "additionalRequirementBanner": "This data connector depends on a parser based on a Kusto Function to work as expected [**CiscoSEGEvent**](https://aka.ms/sentinel-CiscoSEG-parser) which is deployed with the Azure Sentinel Solution.",
+    "graphQueries": [
+        {
+            "metricName": "Total data received",
+            "legend": "CiscoSEG",
+            "baseQuery": "CiscoSEGEvent"
+        }
+    ],
+    "sampleQueries": [
+        {
+            "description" : "Top 10 Source IP",
+            "query": "CiscoSEGEvent\n | where isnotempty(SrcIpAddr)\n    | summarize count() by SrcIpAddr\n | top 10 by count_"
+        }
+    ],
+    "dataTypes": [
+        {
+            "name": "CiscoSEG",

```

Comment:
DataType should be CommonSecurityLog (CiscoSEG) as this is CEF connector, please modify.

Reply:
Looks good. Thanks.

Reply:
updated